src/Persistence/Initialization/CharacterClasses/ClassSummoner.cs

```diff
@@ -98,6 +98,7 @@ private CharacterClass CreateSummoner(CharacterClassNumber number, string name,
 
         result.AttributeCombinations.Add(this.CreateAttributeRelationship(Stats.MinimumCurseBaseDmg, 1.0f / 9, Stats.TotalEnergy));
         result.AttributeCombinations.Add(this.CreateAttributeRelationship(Stats.MaximumCurseBaseDmg, 1.0f / 4, Stats.TotalEnergy));
+        result.AttributeCombinations.Add(this.CreateAttributeRelationship(Stats.CurseAttackDamageIncrease, 1.0f / 100, Stats.BookRise));
```

Comment:
@sven-n , I think because `AttributeDefinition` does not implement `IAssignable` it is this is causing this exception:

![exception](https://github.com/user-attachments/assets/675490cc-aa99-4bff-91d8-ff83030d5be7)
How to get around this?
I've tested  and even after server restart, the `CurseAttackDamageIncrease` does not change when equipping a book with the `BookRise` base power up attribute.